src/Components/Button/index.tsx

```diff
@@ -1,9 +1,12 @@
+import { ForwardedRef, forwardRef } from 'react';
 import { ButtonProps } from 'antd';
 
 import { StyledButton } from './styles';
 
-const Button = ({ type = 'primary', shape = 'round', ...rest }: ButtonProps) => (
-  <StyledButton type={type} shape={shape} {...rest} />
+const Button = forwardRef(
+  ({ type = 'primary', shape = 'round', ...rest }: ButtonProps, ref: ForwardedRef<HTMLButtonElement>) => (
+    <StyledButton type={type} shape={shape} ref={ref} {...rest} />
+  )
```

Comment:
Due to some research I found out that this warning was about missing ref
![image](https://github.com/MendelDamian/plemiona-web/assets/44109041/1af2d6ba-0679-42dc-826a-b852ce603805)